sdk/remoterendering/mixedreality-remoterendering/src/authentication/authenticationEndpoint.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Constructs the authentication endpoint from a Mixed Reality account domain.
+ * @internal
+ * @param accountDomain The Mixed Reality account domain.
+ */
+export function constructAuthenticationEndpointFromDomain(accountDomain: string): string {
+  if (!accountDomain) {
+    throw new Error("Argument cannot be null or empty: 'accountDomain'.");
+  }
```

Comment:
Can we make the type of `accountDomain` more precise if it could be `null` or `undefined`? e.g. `accountDomain: string | null` or `accountDomain?: string`?

Reply:
Yes, if this function is called then accountDomain should be neither null or undefined. Since the typesystem should ensure that, I guess I can remove the check.